# This is not optional since we need to match response structure, we need to also send correct args for it.
HttpbinPostArgs params

# This is optional, or contains only one HttpHeaders message
HttpRequestHeaders[] headers

# This is not optional since we need to match response structure, we need to also send correct args for it.
HttpFormItem[] form

---
#pyros_msgs/HttpStatusCode
uint16 status

# this is not optional : it is always present in response
HttpbinPostArgs args

# this is not optional : it is always present in response
string data

HttpFormItem[] form

# this is not optional : it is always present in response
HttpRequestHeaders headers

string origin
string url
